quickstart_lists/effortless_x86_64-linux_stable

```diff
@@ -0,0 +1,10 @@
+chef/chef-client
```

Comment:
We should also include `x86_84-linux-kernel2` lists, or instructions to use this list, as they are identical.